storage/innobase/include/rem0types.h

```diff
@@ -31,7 +31,7 @@ Created 5/30/1994 Heikki Tuuri
 typedef byte	rec_t;
 
 /** This type represents a field offset in a rec_t* */
-typedef unsigned short int offset_t;
+typedef unsigned short int rem0offset_t;
```

Comment:
Given this name appear in a loooot of places, I believe the name should be as intuitive as possible. Marko suggested `rec_offs`, sounds good enough for me. @kevgs has the final word on this as he did the refactoring.